search-provider/eks-shards-util.c

```diff
@@ -0,0 +1,102 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#include "eks-query-util.h"
+
+#include <eos-knowledge-content.h>
+#include <eos-shard/eos-shard-shard-file.h>
+
+#include <gio/gio.h>
+
+/* For instance on "/frob/foo/bar/baz" "foo", return "bar/baz" */
```

Comment:
Either this comment is wrong, or I don't understand what is "reverse" about this function.

Reply:
It returns the last path components up until some component is found. I'll try and think of a better name.

Reply:
:+1: